app/views/devices_guidance/how_to_order_laptops_for_social_care_leavers.html.erb

```diff
@@ -0,0 +1,209 @@
+    </p>
+
+    <p class="govuk-body">
+      Care leavers cannot order directly.
+    </p>
+
+    <p class="govuk-body">
+      We recommend that you get set up to order as soon as possible to make sure you receive your preferred choice of laptops.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      Which care leavers are eligible for this support
+    </h2>
+
+    <p class="govuk-body">
+      DfE is providing these to support care leavers your LA is responsible for, who:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>are aged between 16 and 25 years old</li>
+      <li>do not have access to a laptop or tablet from your LA, or any other source</li>
+    </ul>
+
+    <p class="govuk-body">
+      LAs can <%= govuk_link_to 'request 4G wireless routers for eligible care leavers', how_to_request_4g_wireless_routers_path %>
+      without fixed broadband too.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      How to get laptops and internet access
+    </h2>
+
+    <h3 class="govuk-heading-m">
+      1. Check your local authority’s laptop allocation
+    </h3>
+
+
+    <p class="govuk-body">
+      DfE has asked each LA to nominate one or more staff members to order laptops and request internet access for care leavers.
+    </p>
+
+    <p class="govuk-body">
+      If you’ve been nominated to order, we’ll email your LA’s estimated laptop allocation to you. If you have not received
+      this email, <%= govuk_link_to 'contact us', support_ticket_path %>.
+    </p>
+
+    <p class="govuk-body">
+      Allocation estimates are based on each local authority's share of the overall care leaver population.
+    </p>
+
+    <p class="govuk-body">
+      You can also request 4G wireless routers for these care leavers if they do not have a fixed broadband connection.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Tell us if your allocation is wrong
+    </h4>
+    <p class="govuk-body">
+      If your estimated allocation is wrong, <%= govuk_link_to 'contact us', support_ticket_path %> and let us know how
+      many laptops you currently need for eligible care leavers. Do not include any personal data, such as their names or
+      dates of birth.
+    </p>
+
+    <p class="govuk-body">
+      We cannot guarantee that we’ll be able to meet all requests for more laptops. We encourage you to order your
+      current allocation anyway so that you can begin distributing devices. We’ll contact you once we’ve reviewed your request.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      2. Check what care leavers need
+    </h3>
+
+    <p class="govuk-body">
+      We recommend working with your care leavers and IT team as soon as possible to find out the following information.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Laptops
+    </h4>
+
+    <p class="govuk-body">To order laptops, you need to find out which:</p>
+
+    <ol class="govuk-list govuk-list--number">
```

Comment:
@aliuk2012 - can this be an unordered list please? The text formatting is correct.

Reply:
Good spot. I thought this was updated